taskana-adapter-camunda-outbox-rest/src/main/resources/taskana-outbox.properties

```diff
@@ -7,12 +7,12 @@ taskana.adapter.outbox.duration.between.task.creation.retries = PT1H
 #taskana.adapter.outbox.datasource.jndi=java:jboss/datasources/ProcessEnginePostgres
 #taskana.adapter.outbox.datasource.jndi=jdbc/ProcessEngine
 
-#taskana.adapter.outbox.datasource.driver=org.postgresql.Driver
-#taskana.adapter.outbox.datasource.url=jdbc:postgresql://localhost:5432/postgres
-#taskana.adapter.outbox.datasource.username=postgres
-#taskana.adapter.outbox.datasource.password=postgres
+taskana.adapter.outbox.datasource.driver=org.postgresql.Driver
```

Comment:
do we need these changes?

Reply:
We need to change the port, the url and the password to comply to those in our postgres docker container.